src/components/Landing.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+const Login = () => (
```

Comment:
Please let the file name be the same as the default export component

Reply:
@ezrogha worked on feedback